### PR TITLE
Removes - for now - torcache.net

### PR DESCRIPTION
**Pull request by fernandog**
also need to cherry pick to master
@labrys

### CODE REVIEW

File: sickrage/providers/GenericProvider.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def __init__(self, name):
             #'http://torcache.net/torrent/{torrent_hash}.torrent',
             'http://thetorrent.org/torrent/{torrent_hash}.torrent',
             'http://itorrents.org/torrent/{torrent_hash}.torrent',
+            'http://torrage.info/torrent.php?h={torrent_hash}.torrent',
```
Comment (fernandog) [inline sickrage/providers/GenericProvider.py]: looks like in torrage we need to click a button to actual download. If we can't bypass, we cant add it

**Comment by fernandog** [comment]
@rarbg are you still working on that feature about caching?

**Comment by medariox** [comment]
We can add 
`https://torrentproject.se/torrent/{torrent_hash}.torrent`

**Comment by rarbg** [comment]
@fernandog Yes , unfortunately im not ready. Ill try to prioritize that task now , meanwhile i applied a temporary fix on our torcache script to up to your remaining ones :)

**Comment by medariox** [comment]
Approved

**Comment by labrys** [comment]
@medariox Make sure you add a PR to a milestone before you merge it.

**Pull request merged**